python/ray/autoscaler/commands.py

```diff
@@ -94,6 +95,35 @@ def teardown_cluster(config_file, yes, workers_only, override_cluster_name):
         nodes = provider.nodes({TAG_RAY_NODE_TYPE: "worker"})
 
 
+def kill_node(config_file, yes, override_cluster_name):
+    """Kills a random Raylet worker."""
+
+    config = yaml.load(open(config_file).read())
+    if override_cluster_name is not None:
+        config["cluster_name"] = override_cluster_name
+    config = _bootstrap_config(config)
+
+    confirm("This will kill a node in your cluster", yes)
+
+    provider = get_node_provider(config["provider"], config["cluster_name"])
+    nodes = provider.nodes({TAG_RAY_NODE_TYPE: "worker"})
+    node = nodes[random.randint(0, len(nodes))]
+    logger.info("Terminating worker {}".format(node))
+    updater = NodeUpdaterProcess(
+        node,
+        config["provider"],
+        config["auth"],
+        config["cluster_name"],
+        config["file_mounts"], [],
+        "",
+        redirect_output=False)
+
+    _exec(updater, "ray stop", False, False)
+
+    time.sleep(5)
```

Comment:
why this sleep?

Reply:
To give the Raylet process some time to exit. Not strictly necessary, but I copied the code for cluster teardown: https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/commands.py#L93